mono/unit-tests/Makefile.am

```diff
@@ -129,10 +144,17 @@ check-local:
 	fi;
 
 clean-local:
-	rm -f callspec.exe callspec.pdb
+	rm -f callspec.exe callspec.pdb 
+	rm -f test-embed.exe test-embed.pdb
 
 %.exe: %.cs
 	$(MCS) -r:$(CLASS)/System.dll -r:$(CLASS)/System.Xml.dll -r:$(CLASS)/System.Core.dll -out:$@ $<
 
+build_mono_embed_test:
+	$(MCS) -r:$(CLASS)/System.dll -r:$(CLASS)/System.Xml.dll -r:$(CLASS)/System.Core.dll -out:test-embed.exe ../../samples/embed/test.cs 
+	
+build_mono_embed_invoke_test:
+	$(MCS) -r:$(CLASS)/System.dll -r:$(CLASS)/System.Xml.dll -r:$(CLASS)/System.Core.dll -out:test-embed-invoke-cs.exe ../../samples/embed/invoke.cs $<
```

Comment:
Same here, I think you can just have a `test-embed-invoke.cs.exe` target that depends on `$(top_srcdir)/samples/embed/invoke.cs`.

Reply:
Done